contracts/helpers/TransferHelper.sol

```diff
@@ -156,8 +242,26 @@ contract TransferHelper is TransferHelperInterface, TokenTransferrer {
                 }
             }
 
-            // Call the conduit and execute bulk transfers.
-            ConduitInterface(conduit).execute(conduitTransfers);
+            // If the external call fails, revert with the conduit's
+            // custom error.
+            try ConduitInterface(conduit).execute(conduitTransfers) {} catch (
```

Comment:
So here we’d compare against the magic value rather than just ignoring the return value

Reply:
the magic value comparison is now back in, see https://github.com/ProjectOpenSea/seaport/pull/516#discussion_r911557187 for why we found the need for extcodehash before calling the conduit.